src/main/java/com/redhat/rhjmc/containerjfr/net/WebServer.java

```diff
@@ -234,14 +233,14 @@ private Response serveClientIndex() {
 
         @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
```

Comment:
Also, I just realized this spotbugs suppression can be now removed. It will be removed in the next commit.

Reply:
Nice, that's always a welcome bonus.